Groupe/g2 a kapala paul alexis

PAlexis36 opened the pull request:
C'est fini.

Commit:
Debut page actualite

PAlexis36 committed:
Maquette actualité final

PAlexis36 committed:
Merge branch 'groupe/G2A' into groupe/G2A-KapalaPaulAlexis

luger95 commented on index.html:
```diff
@@ -279,7 +279,52 @@ <h3 id="competence-titre">BOOSTER VOS COMPÉTENCES</h3>
+                <ol id="liste_actualite">
+                    <li> <img src="assets/img/shutterstock_703607812.png" alt="imageactu"> </li>
+                    <li id="Tag_actualite"> 
+                        <p>Tag</p>  
```
Ici, l'utilisation d'une balise p n'est pas fausse, mais pourrait être remplacer par un span

luger95 reviewed (changes requested):
Je t'ai laissé une suggestion d'amélioration

PAlexis36 committed:
Add files via upload

Page avancé final v2

luger95 commented on assets/css/globalStyle.css:
```diff
@@ -78,39 +78,54 @@ h2 {
 }
 
-#Tag_actualite {
+#Tag_actualite1, #Tag_actualite2, #Tag_actualite3, #Tag_actualite4 {
```
l'utilisation d'une class serait plus adaptée

luger95 commented on assets/css/globalStyle.css:
```diff
@@ -78,39 +78,54 @@ h2 {
 }
 
-#liste_actualite {
+#liste_actualite1, #liste_actualite2, #liste_actualite3, #liste_actualite4 {
```
Un style général est souhaitable exemple .list-style-none

luger95 commented on assets/css/globalStyle.css:
```diff
@@ -78,39 +78,54 @@ h2 {
 }
 
-#Texte {
+#Texte1, #Texte2, #Texte3, #Texte4 {
```
Une class css est plus adapté

luger95 commented on assets/css/globalStyle.css:
```diff
@@ -78,39 +78,54 @@ h2 {
+    color: #000000; 
+}
 
+#Texte1::before, #Texte2::before, #Texte3::before, #Texte4::before {
```
Une class css est plus adapté

luger95 commented on assets/css/globalStyle.css:
```diff
@@ -126,44 +141,47 @@ h2 {
 }
 
-#DateActu1,#DateActu2,#DateActu3,#DateActu4{
+#DateActu1, #DateActu2, #DateActu3, #DateActu4 {
```
Une class css est plus adapté

luger95 commented on assets/css/globalStyle.css:
```diff
@@ -126,44 +141,47 @@ h2 {
 
 
 
+
+
+
```
inutile

PAlexis36 committed:
Add files via upload

Version avec la flèche qui marche correctement + correction des problèmes 28/11 11h36
